### PR TITLE
2021-12-31-nerdl_restaurant_100d_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2021-12-31-nerdl_restaurant_100d_en.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    .setInputCols(["sentence", "token", "embeddings"])\
+    .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, embeddings, nerdl, ner_converter])
+
+text = """2 star restaurants with inside dining and 5 star resturants are in my town. Besides that, Hong Kong restaurant reasonable prices a great lunch spot but open till 2 am, called Passims Kitchen."""
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-31-nerdl_restaurant_100d_en.md:54]: The example is not native English, has several grammatical mistakes. Was it taken from the training or test set? Here you can find a good article about a restaurant review in native English: https://www.sassyhongkong.com/eat-drink-lunch-deals-set-menus-restaurants/

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
The example is not correct English. Attached link where to check for native english examples.

**Review by josejuanmartinez — APPROVED** [review]
Wow it looks amazing now, thank you!

File: docs/_posts/murat-gunay/2021-12-31-nerdl_restaurant_100d_en.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-31-nerdl_restaurant_100d_en.md]: @josejuanmartinez 
Yes you may be right. The text was grammatically not so good but it was a native customer comments from train data set. Sometimes, you know, humans don't obey grammar... For the best model result we generally take text from train set as a best practice.

**Pull request merged**